bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/ui/launch/LaunchConfigurationDelegateLauncher.java

```diff
@@ -118,4 +118,53 @@ private void assertProjectsMatch(ILaunchConfiguration configuration, IProject se
             Trace.getTracer().traceExit(Trace.TRACE_UI);
         }
     }
+
+    /**
+     * Starts dev mode
+     * 
+     * @param iProject The project to process.
+     * @param iConfiguration The configuration for this start.
+     * @param launch The launch associated with this start
+     * @param mode The operation mode type. Run or debug.
+     * 
+     * @throws Exception
+     */
+    public void startDevMode(IProject iProject, ILaunchConfiguration iConfiguration, ILaunch launch, String mode) throws Exception {
```

Comment:
Like I mentioned in my StartAction comment, I think I get and like the refactoring overall.

This maybe should not be public.  Maybe it should be named 'startViaDevModeOperations' or something different than the method name in DevModeOperations

Reply:
`launchDevMode` ?